### PR TITLE
 Enable asar archiving for Electron app resources & Add error handling to electron pack

### PR DESCRIPTION
**Pull request by kaviththiranga**

### CODE REVIEW

File: modules/distribution/index.js [hunk 1]
```diff
@@ -4,46 +4,48 @@ const process = require("process");
...
 		fs.mkdirSync(logsDir);
 	}
-	fs.access(logsDir, fs.W_OK, function(err) {
-		if(err){
-			logger.error("can't write to log folder.");
-		}
-		else{
-			logger = new log("info", fs.createWriteStream(path.join(logsDir, "app.log")));
-		}
-	});
```
Comment (roonyh) [inline modules/distribution/index.js]: indentation here uses tabs no? shall we stick to spaces? otherwise it kind of messes up indentation.

Comment (kaviththiranga) [inline modules/distribution/index.js]: sure, will fix this.

Comment (kaviththiranga) [inline modules/distribution/index.js]: @roonyh fixed all lint issues in dist module.

Comment (roonyh) [inline modules/distribution/index.js]: great! thanks for adding eslint :)

**Pull request merged**